service/src/main/java/bio/terra/catalog/service/DatasetService.java

```diff
@@ -161,4 +195,18 @@ private List<TableMetadata> convertDatarepoTablesToCatalogTables(
                     .hasData(tableModel.getRowCount() > 0))
         .toList();
   }
+
+  private ColumnModel convertDatarepoColumnModelToCatalogColumnModel(
```

Comment:
Can this can be `static`? Declaring it as `static` would make it clearer that this doesn't rely on class state.